lib/instrumenter/src/instrumenter.ts

```diff
@@ -584,8 +584,13 @@ export function instrument<TObj extends Record<string, any>>(
   options: Options = {}
 ): TObj {
   try {
-    // Don't do any instrumentation if not loaded in an iframe.
-    if (global.window.parent === global.window) return obj;
+    // Don't do any instrumentation if not loaded in an iframe and it's not running in a capture.
+    if (global.window.parent === global.window) {
+      const params = new URLSearchParams(global.window.location.search);
+      if (!params.get('interactions')) {
```

Comment:
```suggestion
      if (!params.get('instrument')) {
```
It's not just for interactions.

I'm thinking perhaps we should support the inverse as well. So if you set `instrument=false` then instrumentation will be disabled even if loaded within an iframe. In other words, the `instrument` param is respected (`true` or `false`), and if not specified we'll fall back to the window.parent check.